app/views/home/tos_faq.html.erb

```diff
@@ -413,6 +413,8 @@ Please use our search functions for this rather than creating a separate work.</
 			<h5 class="heading" id="collections_def">What do you mean by "collections"?</h5>
 			<p>Collections are groups of works collected together under one heading. Collections can be fic or art fests, exchanges, 'big bangs' matching artists, authors, and/or podficcers, or other types of creative challenges, as well as simple collections of fanworks chosen by the collection maintainer. <a href="https://archiveofourown.org/archive_faqs/18">Learn more about collections.</a>
 			</p>
+                        <h5 class="heading">What information can a challenge maintainer see about participants?</h5>
+                        <p>A challenge or gift exchange maintainer can see prompts as well as the username and email address that participants use to sign up, in case the maintainer needs to communicate with participants. Other collection maintainers cannot see email addresses by virtue of having a user’s work in their collections.</p>
```

Comment:
Really tiny annoying nitpick: "user’s work" has a smart/curly apostrophe when we use straight '